common/test/acceptance/pages/lms/video/video.py

```diff
@@ -356,9 +356,9 @@ def click_player_button(self, button):
 
         self.q(css=button_selector).first.click()
 
-        button_states = {'play': 'playing', 'pause': 'pause'}
-        if button in button_states:
-            self.wait_for_state(button_states[button])
+        # button_states = {'play': 'playing', 'pause': 'pause'}
+        # if button in button_states:
+        #     self.wait_for_state(button_states[button])
```

Comment:
@jzoldak We expect that the Promise calls `check_func` 5 times per second and `check_func` contains a [logger.debug](https://github.com/edx/edx-platform/blob/anton/fix-flaky-tests/common/test/acceptance/pages/lms/video/video.py#L668), but for some reasons there is no any logs from `2:15` to `2:20`.